pkg/collect/ceph.go

```diff
@@ -215,11 +212,7 @@ func findRookCephToolsPod(ctx context.Context, c *CollectCeph, namespace string)
 		return &pods[0], nil
 	}
 
-	if v.GetBool("debug") {
-		return nil, errors.New("rook ceph tools pod not found")
-	}
-
-	return nil, nil
+	return nil, &types.NotFoundError{Name: "rook ceph tools pod"}
```

Comment:
You can just log that the pod was not found here using `logger` package. The log message will be silenced if `--debug` flag was not set. No need to return an error

Reply:
We have `logger.SetQuiet(v.GetBool("quiet"))` in `cmd/troubleshoot/cli/root.go`, however, I didn't see any flag for quiet in support-bundle, so I have changed it to `logger.SetQuiet(!v.GetBool("debug"))`